src/imports.rs

```diff
@@ -39,6 +35,31 @@ impl<'a> FmtVisitor<'a> {
             _ => ""
         };
 
+        if path_list.len() == 1 {
+            let vpi = path_list[0];
+            let rest = match vpi.node {
+                ast::PathListItem_::PathListIdent{ name, .. } => {
+                    token::get_ident(name).to_string()
+                }
+                ast::PathListItem_::PathListMod{ .. } => {
```

Comment:
it would be better to use an `if let ... { return ... }` idiom than a match idiom for these cases, I think that would make this code simple.